packages/relay/src/lib/eth.ts

```diff
@@ -417,32 +479,55 @@ export class EthImpl implements Eth {
     this.logger.trace('call(hash=%o, blockParam=%s)', call, blockParam);
     // The "to" address must always be 42 chars.
     if (call.to.length != 42) {
-      throw new Error("Invalid Contract Address: '" + call.to + "'. Expected length of 42 chars but was" + call.to.length);
+      throw new Error(
+        "Invalid Contract Address: '" +
+          call.to +
+          "'. Expected length of 42 chars but was" +
+          call.to.length
+      );
     }
 
     try {
       // Get a reasonable value for "gas" if it is not specified.
-      const gas: number = call.gas == null
-          ? 400_000
-          : typeof call.gas === 'string' ? Number(call.gas) : call.gas;
+      let gas: number;
+      if (typeof call.gas === 'string') {
+        gas = Number(call.gas);
+      } else {
+        if (call.gas == null) {
+          gas = 400_000;
```

Comment:
We should make this configureable i.e. add to the env file.
So should a change be needed in production for some reason it doesn't require re-deployments

Reply:
in another pr.  I was dodging sonar complaints here.

Reply:
Sounds good